cssdb.json

```diff
@@ -94,18 +90,7 @@
     "description": "Properties for defining the break behavior between and within boxes",
     "specification": "https://www.w3.org/TR/css-break-3/#breaking-controls",
     "stage": 3,
-    "browser_support": {
-      "and_uc": "12.12",
-      "edge": "12",
-      "ie": "10",
-      "ie_mob": "10",
-      "ios_saf": "10",
-      "op_mini": "all",
-      "op_mob": "11.1",
-      "opera": "11.1",
-      "safari": "10",
-      "samsung": "5"
-    },
+    "browser_support": {},
```

Comment:
Why did it only have `"vendors_implementations": 1` before this change?
And is it correct that no browser supports this?

Reply:
https://caniuse.com/mdn-css_properties_break-before_paged_context_recto

No support for some values.